File: cases/taint/go/local-chain-negative/local_chain_negative.go
package dataflowbench

func dfb_source() int { // DFB-SOURCE: local-chain-negative-input
	return 1
}

func dfb_sink(value int) {} // DFB-SINK: local-chain-negative-sink

func run() {
	first := dfb_source()
	second := first // DFB-WITNESS: local-chain-negative-second
	third := second // DFB-WITNESS: local-chain-negative-third
	_ = third
	dfb_sink(0)
}

File: cases/taint/csharp/local-chain-negative/LocalChainNegative.cs
namespace DataFlowBench;

static class LocalChainNegative
{
    static int dfb_source() // DFB-SOURCE: local-chain-negative-input
    {
        return 1;
    }

    static void dfb_sink(int value) { } // DFB-SINK: local-chain-negative-sink

    static void Run()
    {
        int first = dfb_source();
        int second = first; // DFB-WITNESS: local-chain-negative-second
        int third = second; // DFB-WITNESS: local-chain-negative-third
        dfb_sink(0);
    }
}
